run.sh

```diff
@@ -1,3 +1,20 @@
 #!/bin/bash
 
-docker run --rm -it -v $(pwd):/app-dir/local -w /app-dir/local contacts-fixer sh
+# Choose running machine name
+unameOut="$(uname -s)"
+case "${unameOut}" in
+    Linux*)     machine=Linux;;
+    Darwin*)    machine=Mac;;
+    CYGWIN*)    machine=Cygwin;;
+    MINGW*)     machine=MinGw;;
+    *)          machine="UNKNOWN:${unameOut}"
+esac
+
+# Mount repository path
+if [ $machine == "Linux" ]
+then
+	docker run --rm -it -v $(pwd):/app-dir/local -w /app-dir/local contacts-fixer sh
+else
+	windows_path=$(pwd | sed 's/^\///' | sed 's/\//\\/g' | sed 's/^./\0:/')
```

Comment:
Is there a cleaner way to do that? It seems that there should be a cleaner way (without sed magic)

Reply:
Started by reducing the number of calls to sed command.

Reply:
Fixed